drivers/ThinQ_Fridge.groovy

```diff
@@ -116,6 +118,7 @@ def processStateData(data) {
       else if (getTemperatureScale() == "F" && device.getDataValue("tempUnit") == "CELSIUS")
         temp = celsiusToFahrenheit(temp)
       sendEvent(name: "freezerTemp", value: temp)
+      sendEvent(name: "freezerTemp°", value: temp + "°" + getTemperatureScale())
```

Comment:
I agree with the idea of adding the unit, but the correct way to do this in HE is by adding the unit to the event such as `sendEvent(name: "freezerTemp", value: temp, unit: "°"+ getTemperatureScale())`

Reply:
Your call...  I was adding a NEW string variable (freezerTemp°) to the devices' attributes which contained the devices' temperature scale display units for use in ones HE dashboard display or text messages.  The original freezerTemp numeric value is great for using in WebCore or RM logic but lacks scale units for text/TTS messages or dashboard display unless one adds a CSS to their Tile#.  Just making a new pretty attribute cause I personally dislike using a tile CSS on dashboard displays...  Probably just my individual programming style...  No problem...

Reply:
I didn't realize until just now that setting a unit will not give you a way to display that unit on an attribute tile. I really hate the HE dashboards :( I stopped using them a long time ago because they were so limiting. I'll add attributes like this but I'm going to rename them to `freezerTempDisplay` only reason is then it is consistent with other drivers I've written.